content/applications/finance/accounting/customer_invoices/customer_addresses.rst

```diff
@@ -21,6 +21,10 @@ On quotations and sales orders, there are now fields for :guilabel:`Invoice Addr
 contact record, the corresponding field will use that address by default, but any contact's address
 can be used instead.
 
+.. seealso::
+  For more information, refer to the documentation on :ref:`Contact Form Configuration
+  <send_quotations/contact-form-config>`
+
```

Comment:
Suggestion to add an inline reference on the contact record too. I think the seealso is a nice add as well. Also needs a period at the end, and to be indented by another space :)
```suggestion
:ref:`contact record <sales/send_quotations/contact-form-config>`, the corresponding field will use
that address by default, but any contact's address can be used instead.

.. seealso::
   For more information, refer to the documentation on :ref:`Contact Form Configuration
   <sales/send_quotations/contact-form-config>`.
```